src/predicates.js

```diff
@@ -0,0 +1,129 @@
+import base64 from './ponyfills/base64.js';
+
+const AND = 'and';
+const OR = 'or';
+
+const FILTER_OPERATORS = {
+  contains: function(a, b) { return a.indexOf(b) >= 0; },
+  equal: function(a, b) { return a === b; },
+  exists: function(a) { return a !== null; }, // Check that the key exists in the dictionary object
+  not_contains: function(a, b) { return !(a.indexOf(b) >= 0); },
+  not_equal: function(a, b) { return a !== b; },
+  not_regex_match: function(value, pattern) { return value && !value.match(pattern); },
+  not_regex64_match: function(value, pattern) { return !regex64Match(value, pattern) },
+  not_starts_with: function(a, b) { return !a.startsWith(b); },
+  kv_contains: (obj, params) => (params[0] in obj) && (obj[params[0]].indexOf(params[1]) >= 0),
```

Comment:
Arrow functions... case in point for transpiler.
```suggestion
  kv_contains: function(obj, params) { return (params[0] in obj) && (obj[params[0]].indexOf(params[1]) >= 0; },
```

Reply:
Thanks, those operators were removed and then added back for testing. They'll be excluded from the final library.